### ISSUE TITLE
Basic Homepage - UI

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#25]
## User story
1. As a user
2. I want / need to see a homepage when accessing deskstar
3. So that I can log in or register

## Acceptance criteria
* Homepage should give access to the login and register function
* Homepage should include the titel or logo of the project
* Only frontend part

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
The work for that issue can be found in this branch:
https://github.com/amosproj/amos2022ws05-shared-desk-mgmt/tree/homepage-ui-and-auth
PR (cannot add it here)
https://github.com/amosproj/amos2022ws05-shared-desk-mgmt/pull/4
